migrate.js

```diff
@@ -313,10 +314,20 @@ exports.init = function init(sbot, config) {
 
         if (config.db2.dangerouslyKillFlumeWhenMigrated) {
           rimraf(flumePath(config.path), (err) => {
-            if (!err) oldLogExists.set(false)
-            doneMigrating()
+            if (err) return console.error(err)
+            if (sbot.db) {
+              sbot.db.loadStateFeeds(() => {
+                sbot.db.setStateFeedsReady(true)
+                oldLogExists.set(false)
+                doneMigrating()
+              })
+            } else {
+              oldLogExists.set(false)
+              doneMigrating()
+            }
           })
         } else {
+          if (sbot.db) sbot.db.loadStateFeeds()
```

Comment:
Shouldn't this line also do a `sbot.db.setStateFeedsReady(true)` like above. For me this looks like the case where you are migrating but don't have kill log enabled?

Reply:
Oops, that's correct, will fix.